python/sources/web_api_gateway/README.md

```diff
@@ -4,10 +4,14 @@
 
 ## How to run
 
-Create a [Quix](https://portal.platform.quix.ai/self-sign-up?xlink=github) account or log-in and visit the Samples to use this project.
```

Comment:
We normally have these links because if the person comes direct to github they dont know what to do or where to use this.